### PR TITLE
Individuals sighting table using cell renderers

### PR DESCRIPTION
**Pull request by brmscheiner**

### CODE REVIEW

File: src/components/cards/SightingsCard.jsx [hunk 1]
```diff
@@ -73,51 +57,15 @@ export default function SightingsCard({
...
-            >
-              {get(
-                sightingObj,
-                'formattedLocation',
-                intl.formatMessage({ id: 'LOCATION_NAME_MISSING' }),
-              )}
-            </Button>
-          );
-        },
+        cellRenderer: cellRendererTypes.location,
```
Comment (brmscheiner) [inline src/components/cards/SightingsCard.jsx:68]: This PR adds the ability to specify a "cell renderer" when setting up a table. It also provides the first handful of renderers, which are meant to abstract logic that was being repeated in multiple tables. This diff here is the goal: replace complicated customBodyRender functions that are repeated in multiple places with a simple config string

File: src/components/dataDisplays/CollapsibleRow.jsx [hunk 2]
```diff
@@ -42,11 +40,22 @@ export default function CollabsibleRow({
...
+            c,
+            'options.cellRendererProps',
+            {},
+          );
+          const customCellRenderer = get(
             c,
             'options.customBodyRender',
-            defaultCellRenderer,
           );
+          const RequestedCellRenderer =
```
Comment (Atticus29) [inline src/components/dataDisplays/CollapsibleRow.jsx:57]: why the capital letter?

File: src/components/dataDisplays/CollapsibleRow.jsx [hunk 3]
```diff
@@ -57,7 +66,15 @@ export default function CollabsibleRow({
                 ...cellStyles,
               }}
             >
-              {cellRenderer(cellValue, datum)}
+              {customCellRenderer ? (
+                customCellRenderer(cellValue, datum)
+              ) : (
+                <RequestedCellRenderer
```
Comment (Atticus29) [inline src/components/dataDisplays/CollapsibleRow.jsx:72]: There's something that I might not understand here. How is this a component??

File: src/components/dataDisplays/CollapsibleRow.jsx [hunk 4]
```diff
@@ -8,16 +8,14 @@ import DownIcon from '@material-ui/icons/KeyboardArrowDown';
 import UpIcon from '@material-ui/icons/KeyboardArrowUp';
 import Collapse from '@material-ui/core/Collapse';
 
+import { cellRenderers } from './cellRenderers';
```
Comment (Atticus29) [inline src/components/dataDisplays/CollapsibleRow.jsx:11]: Did not know that you could import multiple files this way. Very cool.

File: src/components/dataDisplays/CollapsibleRow.jsx (hunk 3 above)
Comment (Atticus29) [reply src/components/dataDisplays/CollapsibleRow.jsx:72]: Nvm. I get it now. The renderers imported from the renderers directory are the components.

File: src/components/dataDisplays/CollapsibleRow.jsx (hunk 2 above)
Comment (Atticus29) [reply src/components/dataDisplays/CollapsibleRow.jsx:57]: Ignore. I get it now.

File: src/components/dataDisplays/cellRenderers/index.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import DefaultRenderer from './DefaultRenderer';
```
Comment (Atticus29) [inline src/components/dataDisplays/cellRenderers/index.js:1]: I don't understand why this file needs to be an index.js. Is that important?

Comment (Atticus29) [reply src/components/dataDisplays/cellRenderers/index.js:1]: Maybe you _can't_ import multiple files as I stated above. It's just that importing a directory looks for index.js if no file is named explicitly?

**Review by Atticus29 — APPROVED** [review]
LGTM. Mostly just educational questions I've asked here.

File: src/components/dataDisplays/cellRenderers/index.js (hunk 5 above)
Comment (brmscheiner) [reply src/components/dataDisplays/cellRenderers/index.js:1]: Exactly!

**Pull request merged**